docker/Dockerfile

```diff
@@ -8,28 +8,41 @@ ARG TRAVIS
 ARG TRAVIS_BRANCH
 
 COPY flecsi-third-party/ /home/flecsi/tpl
+RUN rm -rf /home/flecsi/.ccache
 COPY ccache/ /home/flecsi/.ccache
 USER root
-RUN chown -R flecsi:flecsi /home/flecsi/tpl /home/flecsi/.ccache /usr/local
+RUN chown -R flecsi:flecsi /home/flecsi/tpl /home/flecsi/.ccache
 USER flecsi
 
 WORKDIR /home/flecsi/tpl
-RUN mkdir -p build && cd build && \
-    cmake -DBUILD_SHARED_LIBS=ON \
+RUN mkdir build
+RUN mkdir install
+WORKDIR build
+RUN ccache -z
+RUN cmake -DBUILD_SHARED_LIBS=ON \
           -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} \
           -DUSE_SYSTEM_LIBS=${SYSTEM_LIBS} \
-          .. && make -j2;
+          -DCMAKE_INSTALL_PREFIX:PATH=/home/flecsi/tpl/install/ \
```

Comment:
Why are we installing it in `/usr/local` anymore?

Reply:
I thought this is easier. Was there a particular reason we want to install in usr/local?

Reply:
`/usr/local` is in CMake's default search path, so we don't need to specify the path (e.g. in `Legion_ROOT`) in the dependent containers.